ext/phar/stream.c

```diff
@@ -44,7 +44,11 @@ php_stream_wrapper_ops phar_stream_wops = {
 	phar_wrapper_unlink,   /* unlink */
 	phar_wrapper_rename,   /* rename */
 	phar_wrapper_mkdir,    /* create directory */
-	phar_wrapper_rmdir,    /* remove directory */
+	phar_wrapper_rmdir     /* remove directory */
+#ifdef PHP_STREAMS_SUPPORT_IS_CACHEABLE
```

Comment:
I agree, having this define looks pointless - if we accept this RFC, this is part of PHP 7 and should be checked against as "is this PHP 7?" if somebody needs to maintain cross-version extensions. We don't have separate defines for every PHP 7 feature, and it's not configurable in any way, so it should be just there - or, if version portability is needed (not sure for phar, it's a core module, why would it need that?) use PHP version checks.